libc/test/UnitTest/LibcTest.cpp

```diff
@@ -197,9 +203,11 @@ int Test::runTests(const TestOptions &Options) {
   }
 
   if (TestCount > 0) {
-    tlog << "Ran " << TestCount << " tests. "
-         << " PASS: " << TestCount - FailCount << ' ' << " FAIL: " << FailCount
-         << '\n';
+    if (!Options.Brief) {
+      tlog << "Ran " << TestCount << " tests. "
+           << " PASS: " << TestCount - FailCount << ' ' << " FAIL: " << FailCount
+           << '\n';
+    }
```

Comment:
at this point, probably should just make this `if (TestCount == 0) tlog << "No Tests run.\n" ...`.